File: python_programming_level2/classes_and_objects/generateprimes.c
// generate prime numbers from 1 to 100

#include <stdio.h>
#include <stdbool.h>

int main(void)
{
    int p, d;
    bool isPrime;

    for (p = 2; p <= 100; p++)
    {
        isPrime = true;

        for (d = 2; d < p; d++)
        {
            if (p % d == 0)
            {
                isPrime = false;
            }
        }

        if (isPrime != false)
        {
            printf("%i ", p);
        }
    }

    printf("\n");

    return 0;
}

File: python-programming-workshop/data_structures/list/generate-primes.py
# generate prime numbers between 1 and 100
# prime numbers are numbers that are divisible by 1 and itself
# 1 is not a prime number
def generate_primes():
    primes = []
    for num in range(2, 100):
        for i in range(2, num):
            if num % i == 0:
                break
        else:
            primes.append(num)
    return primes

print(generate_primes())
